src/frontend/apps/lti_site/components/DashboardVOD/index.tsx

```diff
@@ -64,8 +70,11 @@ export const DashboardVOD = () => {
           startDate={video.starting_at}
         />
       </StyledLiveVideoInformationBarWrapper>
-
-      <VideoWidgetProvider isLive={false} isTeacher />
+      {video.live_state === liveState.ENDED ? (
+        <DashboardControlPane isLive={false} />
+      ) : (
+        <VideoWidgetProvider isTeacher isLive={false} />
+      )}
```

Comment:
Reading like this is a little bit disturbing. Why not computing the `isLive` props value before, calling the variable with something like `fromLive` or something more accurate and then use it without ternary condition ?